spec/page-object/loads_platform_spec.rb

```diff
@@ -34,19 +34,19 @@ class TestLoadsPlatform
   end
 
   context "When an unknow object is passed in" do
-    let(:basic_message) { 'Unable to pick a platform for the provided browser.' }
```

Comment:
I changed messages to indicate that elements are possible options for a page object, not just browsers.